### DISCUSSION TITLE
Unable to get allure environment details in report while using junit5

### DISCUSSION THREAD

**Discussion by Vishnu-2810** [allure-framework/allure2#2046] [Questions & Support]
Unable to get allure environment details in allure report 
While running it in terminal getting that allure environment is saved but not displayed
Using JUnit 5 @baev 

Please help to fix this ,
Thanks in advance

**Comment by cheshi-mantu** [comment]
@Vishnu-2810 hi there!
could you please provide a bit more details? what is it you are trying to achieve? What is it you are doing for that? Please provide some examples for us to get the context.

cheers
